templates/etc/signup.html

```diff
@@ -12,19 +12,31 @@
     $ captcha_public_key = CAPTCHA()
 
     $if captcha_public_key:
+      <label for="g-recaptcha-response">Captcha</label>
+      <script src="https://www.google.com/recaptcha/api.js" async defer></script>
+      <div class="g-recaptcha" data-sitekey="${captcha_public_key}"></div>
```

Comment:
I haven't tried it locally myself yet, but `data-theme="dark"` will probably look better than the light theme. Screenshots of both might be good.